include/deal.II/lac/arpack_solver.h

```diff
@@ -162,6 +162,14 @@ class ArpackSolver : public Subscriptor
   void set_initial_vector(const VectorType &vec);
 
   /**
+   * Set real @p sigmar and complex @p sigmai parts of the shift for
+   * shift-and-invert spectral transformation.
+   *
+   * If this function is not called, the shift is assumed to be zero.
+   */
+  void set_shift(const double sigmar, const double sigmai = 0);
```

Comment:
Can you wrap the two arguments into separate lines?

I thought about making the argument a `std::complex<double>`. I realize that you want the default value for the imaginary part to be zero. But what happens if you had
```
  void set_shift (const std::complex<double> &sigma);
```
and then did
```
  arpack_solver.set_shift (3.141);
```
? Does it automatically convert the double argument to a complex value with zero imaginary part?